api/docs/v2/storefront/index.yaml

```diff
@@ -3145,6 +3168,143 @@ components:
           example: New York
           description: State name
       x-internal: true
+    Store:
+      type: object
+      description: 'Stores are the center of the Spree ecosystem. Each Spree installation can have multiple Stores. Each Store operates on a different domain or subdomain.'
+      title: Store
+      x-internal: true
+      properties:
+        id:
+          type: string
+          example: '1'
+        type:
+          type: string
+          default: store
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              example: Spree Demo
+              description: Name of the Store
+            url:
+              type: string
+              example: spree-demo.com
+              description: Store Url
+            meta_description:
+              type: string
+              example: Luxury clothing store for men and women.
+              description: Meta description for SEO
+            meta_keywords:
+              type: string
+              example: luxury clothing men women
+              description: Meta keywords for SEO
+            default_currency:
+              type: string
+              example: EUR
+              description: Default currency of the store
+            supported_currencies:
+              type: string
+              example: EUR,USD,GBP
+              description: All currencies supported by store
+            facebook:
+              type: string
+              example: https://www.facebook.com/mystorename
+              description: URL of Facebook page
+            twitter:
+              type: string
+              example: https://twitter.com/mystorename
+              description: URL of Twitter page
+            instagram:
+              type: string
+              example: https://instagram.com/mystorename
+              description: URL of Instagram page
+            default_locale:
+              type: string
+              example: en
+              description: Default locale of the Store
+            supported_locales:
+              type: string
+              example: en,es,de
+              description: Supported locales of the Store
+            customer_support_email:
+              type: string
+              example: support@mystore.com
+              description: This email is visible to your Store visitors in the Footer section
+            default_country_id:
```

Comment:
@aleksandarpetrushev this isn't right, default country is an association, not an ID column in the serializer

Reply:
@aleksandarpetrushev let's prepare another PR with removal of that attribute